app/helpers/ops_helper/group_rbac_details_helper.rb

```diff
@@ -17,6 +17,27 @@ def group_rbac_details_summary(group)
     safe_join(summary)
   end
 
+  def select_tree_node(treeId)
```

Comment:
It might be easier to read if these methods had the work onclick in the method name. Something like `select_tree_node_onclick` or `onclick_select_tree_node`

Reply:
Actually i see you did `_onclick` in the other places, so let's do that consistently across all the new methods.